platform/core/Makefile

```diff
@@ -181,8 +181,8 @@ endef
 
 # ============ Main targets ============ #
 
-.PHONY: rust-core
-rust-core: $(target)
+.PHONY: rust_core
```

Comment:
You're normalizing to snake_case all targets, right?

Reply:
Yes.

Reply:
Good call.